### DISCUSSION TITLE
Extensions issue

### DISCUSSION THREAD

**Discussion by JohnRamanujan** [coder/code-server#5092] [General]
extensions which are installed through the cli, not showing in UI and even not working  . .
![Screenshot 2022-04-13 at 1 20 26 PM](https://user-images.githubusercontent.com/96564316/163127069-904c8cda-d374-47ac-b2a9-7d1dc62f1266.jpg)
![Screenshot 2022-04-13 at 1 21 56 PM](https://user-images.githubusercontent.com/96564316/163127378-64cc89ab-0ca4-4ee8-b347-c61fb8520c07.jpg)
![Screenshot 2022-04-13 at 1 22 49 PM](https://user-images.githubusercontent.com/96564316/163127534-b5d8e4a4-84a6-44b1-81c3-ea23b31fb54a.jpg)

**Comment by lg-nineoneninethreenine** [comment]
#weird x-infinity.
Wait you installed a theme! Reload your instance since the server has to load the extension into the extension list. It might fix the job since code-server sometimes slows down in this case.
Edit: wait try using the normal startup for code-server (`exec code-server`)

**Reply by jsjoeio** [reply]
Did either of these work for you?
